src/main/scala/cromwell/binding/package.scala

```diff
@@ -21,8 +21,9 @@ package object binding {
   type WorkflowCoercedInputs = Map[FullyQualifiedName, WdlValue]
   type WorkflowOutputs = Map[FullyQualifiedName, WdlValue]
   type FullyQualifiedName = String
+  type LocallyQualifiedName = String
```

Comment:
I agree with the move to case classes, though perhaps that could be a ticket to replace them consistently.